mlflow/server/js/src/experiment-tracking/components/run-page/overview/RunViewParamsTable.test.tsx

```diff
@@ -0,0 +1,57 @@
+import { keyBy } from 'lodash';
+import { renderWithIntl, fastFillInput, act, screen } from 'common/utils/TestUtils.react17';
+import { KeyValueEntity } from '../../../types';
+import { RunViewParamsTable } from './RunViewParamsTable';
+import { MemoryRouter } from '../../../../common/utils/RoutingUtils';
+
+const testRunUuid = 'test-run-uuid';
+
+// set a longer timeout so this passes on CI
+jest.setTimeout(10000);
```

Comment:
cc @hubertzub-db i added this line which seems to help it pass in CI. it was timing out waiting 5 seconds, but after adding this it succeeds.

Reply:
@daniellok-db 
I don't see any problem, actually [we have a lot of tests with increased timeout](https://src.dev.databricks.com/search?q=context%3Aglobal+jest.setTimeout%28&patternType=standard&sm=1&groupBy=repo) due to RTL being treated as integration-ish tests which inherently can need more time. I'd just 
- improve justification to, maybe `// Larger timeout for integration testing (table rendering)`
- set same timeout in `RunViewMetricsTable.test.tsx` since it's almost identical test